nextflow.config

```diff
@@ -8,8 +8,15 @@
 // Global default params, used in configs
 params {
 
+  // Workflow flags
+  // TODO nf-core: Specify your pipeline's command line flags
+  input = null
+  input_paths = null
```

Comment:
I don't think that your pipeline uses this - typically needed for the test confg.